bokeh/tests/test_properties.py

```diff
@@ -317,14 +317,28 @@ class IsDelegate(HasProps):
 
         class IncludesDelegateWithPrefix(HasProps):
             z = Include(IsDelegate, use_prefix=True)
-            z_y = String("world") # override the Include
+            z_y = Int(57) # override the Include
 
         class IncludesDelegateWithoutPrefix(HasProps):
             z = Include(IsDelegate, use_prefix=False)
-            y = String("world") # override the Include
+            y = Int(42) # override the Include
+
+        class IncludesDelegateWithoutPrefixUsingOverride(HasProps):
+            z = Include(IsDelegate, use_prefix=False)
+            y = Override(default="world") # override the Include changing just the default
```

Comment:
Whoops, wrote my previous comment before seeing this one!

My original intent for Include was definitely as more of a macro.  Perhaps I am not fully conceptualizing what's possible with mixins, but my familiarity with them from an AOP or "traditional static OOP language" perspective is that they are not really parameterized at class generation time (not to be confused with instance generation).

The problem that I was attempting to solve is actually deeply related to the theming question.  Basically, there are many components in a plot that have "typed subcomponents".  A basic Axis has two different Text things (tick labels, axis labels) and three different Line things (tick_in, tick_out, axis_line).

I suppose I could have modeled this differently, by using Has-A instead of Is-A, and actually compose a sub-graph of first-class subcomponents in the Axis.  Then, the user would type:

    a = Axis()
    a.axis_line.line_dash = "dotted"
    a.label.font = "comic sans bold 16pt"

But from an "ease of reading" perspective and some other reasons I can't quite remember now, I chose to do the macro-based approach.